global.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="@docusaurus/theme-classic" />
```

Comment:
that works, but my advice was rather to do this instead and improve typesafety of your config file:

```ts
import type {Config} from '@docusaurus/types';
import type * as Preset from '@docusaurus/preset-classic';

const config: Config = {
  title: 'My Site',
  presets: [
    [
      'classic',
      {
        docs: {
          
        },
        blog: {
          
        },
      } satisfies Preset.Options,
    ],
  ],
};
```